Cargo.toml

```
@@ -78,6 +81,7 @@ rkyv = ["dep:rkyv"]
 rkyv-safe = ["dep:bytecheck", "rkyv/validation"]
 rocket-traits = ["dep:rocket"]
 rust-fuzz = ["dep:arbitrary"]
+scale-codec =  ["parity-scale-codec-derive","parity-scale-codec","scale-info"]
```

We're starting to use the updated format for feature dependencies:
```suggestion
scale-codec =  ["dep:parity-scale-codec-derive","dep:parity-scale-codec","dep:scale-info"]
```